guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -144,16 +188,34 @@ endif::[]
 ----
 # leapp preupgrade
 ----
-The first run is expected to fail but report issues and inhibit the upgrade.
+ifdef::satellite[]
++
+If you run {Project} in a disconnected environment, add the `--no-rhsm` and `--enablerepo` parameters:
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# leapp preupgrade \
+--no-rhsm \
+--enablerepo BaseOS \
+--enablerepo AppStream \
+--enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+----
++
+When running `leapp preupgrade` and `leapp upgrade`, you must pass `--enablerepo <id>` for each manually defined repository from the previous step.
```

Comment:
Not sure what to put in place of `<id>`, but I think there should be another sentence like "Replace `<id>` with the id of the repository." I don't know whether it is an id of the repository, but it should be mention what to replace `<id>` with.

Reply:
I believe the `<id>` is for the repository ID. I feel like the `<id>` is self explanatory for the user to put the ID number of the repository here. Would an extra sentence explaining it as suggested make sense here @evgeni ?

Reply:
For hightlighting, we generally use italics if users have to make adjustements, so that'd be `_ID_` in asciidoc.

Reply:
I'd argue this line is a duplicate of the explicit instructions at the beginning of this step and can be dropped alltogether?

However, I'd add the same explicit commands as above to the `leapp upgrade` invocation below.

Reply:
Sure, if you can simplify steps by removing parts, go for it!

Reply:
I went ahead and removed the line and took the commands above and moved them into the step after running `leapp upgrade`.